packages/compartment-mapper/test/test-cjs-compat.js

```diff
@@ -0,0 +1,23 @@
+// import "./ses-lockdown.js";
+import 'ses';
```

Comment:
not sure why these are here, I cargo-culted them in.

Reply:
This could be:

```js
import '@endo/init';
```

If we take a dependency on init, which we probably should now.

Reply:
switching the import explodes for me, I might look into it more.

For now, I got the best results doing 
```js
import '../../ses/index.js';
``` 
because I get nicer stacktraces